src/Features/Core/Portable/ExtractInterface/AbstractExtractInterfaceService.cs

```diff
@@ -61,7 +60,11 @@ public async Task<ImmutableArray<ExtractInterfaceCodeAction>> GetExtractInterfac
             Action<string, NotificationSeverity> errorHandler,
             CancellationToken cancellationToken)
         {
-            var typeAnalysisResult = AnalyzeTypeAtPositionAsync(documentWithTypeToExtractFrom, position, TypeDiscoveryRule.TypeDeclaration, cancellationToken).WaitAndGetResult_CanCallOnBackground(cancellationToken);
+            var typeAnalysisResult = AnalyzeTypeAtPositionAsync(
+                documentWithTypeToExtractFrom,
+                position,
+                TypeDiscoveryRule.TypeDeclaration,
+                cancellationToken).WaitAndGetResult_CanCallOnBackground(cancellationToken);
```

Comment:
Nothing changed here, right? Just formatting?

Also, shouldn't the containing method be updated to accept the argument for using the same file?

Reply:
Using the same file is determined by the ExtractInterfaceOptions, which in VS invokes the dialog. I don't think the containing method should have a preconceived notion of the location beforehand.

Reply:
is there a reason why this use WaitAndGetResult_xxx rather than just return Task<…> and caller of this or the very top caller on UI call it with WaitAndGetResult?